features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/site/public/locales/en.json

```diff
@@ -93,7 +94,8 @@
   "Apis.Details.Overview.subscribe.to.application.btn": "Subscribe to an Application",
   "Apis.Details.Overview.subscriptions": "Subscriptions >>",
   "Apis.Details.Overview.view.credentials": "View Credentials",
-  "Apis.Details.Sdk.no.lanuages": "No languages are configured.",
+  "Apis.Details.Sdk.no.sdks": "No SDKs Yet",
+  "Apis.Details.Sdk.no.sdks.content": "No SDKs available for this API yet",
```

Comment:
```suggestion
  "Apis.Details.Sdk.no.sdks.content": "No SDKs available for this API",
```

Reply:
Thanks, the suggested changes have been made and merge conflicts resolved.